src/main/app-launcher.ts

```diff
@@ -42,23 +42,57 @@ let findAppsAndPrefs = async () => {
   }
 }
 let createChoices = async () => {
-  let { extractIcon } = await npm("get-app-icon")
+  let extractIcon =
+    process.platform === "win32"
+      ? (await npm("get-app-icon")).extractIcon
+      : () => Promise.resolve(undefined)
```

Comment:
I define this up here instead of the loop below for performance and I return a function that returns a promise when on other platforms because we `.catch` down below.